packages/strapi-admin/services/user.js

```diff
@@ -194,6 +194,26 @@ const deleteById = async id => {
   return strapi.query('user', 'admin').delete({ id });
 };
 
+/** Delete a user
+ * @param ids ids of the users to delete
+ * @returns {Promise<user>}
+ */
+const deleteByIds = async ids => {
+  // Check at least one super admin remains
+  const usersToDelete = await strapi.query('user', 'admin').find({ id_in: ids }, ['roles']);
```

Comment:
I think we can simplify this with count here.

count nbr of users with role admin.
count nbr of users with role admin and id_in ids.

if same count throw error.  this would avoid loading the roles of all those users.